tests/conftest.py

```diff
@@ -1,29 +1,35 @@
 import pytest
-from brownie import ZERO_ADDRESS, RewardsManager, FarmingRewards, Mooniswap, MooniswapFactoryGovernance
-from utils.config import ldo_token_address, steth_token_address, dai_address, one_inch_token_address, rewards_amount, scale
+from brownie import ZERO_ADDRESS, RewardsManager
+from utils.config import (
+    gift_index,
+    ldo_token_address,
+    steth_token_address,
+    dai_address,
+    one_inch_token_address,
+    lido_dao_agent_address,
+    farming_rewards_address,
+    farming_rewards_owner,
+    mooniswap_address
+)
 
 
 @pytest.fixture(scope="function")
-def rewards_manager(ape, farming_rewards, ldo_token):
-    rewards_manager_contract = RewardsManager.deploy(farming_rewards, {"from": ape})
-    assert farming_rewards.addGift(ldo_token, rewards_amount, rewards_manager_contract, scale, {"from": ape})
-    return rewards_manager_contract
+def rewards_manager(ape, farming_rewards):
+    manager = RewardsManager.deploy(farming_rewards, {"from": ape})
 
+    farming_rewards.setRewardDistribution(gift_index, manager, {"from": farming_rewards_owner})
 
-@pytest.fixture(scope="function")
-def mooniswap_factory(ape):
-    return MooniswapFactoryGovernance.deploy(ZERO_ADDRESS, {"from": ape})
+    return manager
 
 
 @pytest.fixture(scope="function")
-def mooniswap(ape, mooniswap_factory):
-    return Mooniswap.deploy(steth_token_address, dai_address, "stETH-DAI Liquidity Pool Token", "LP", mooniswap_factory, {"from": ape})
+def mooniswap(interface):
+    return interface.Mooniswap(mooniswap_address)
 
 
 @pytest.fixture(scope="function")
-def farming_rewards(ape, mooniswap, one_inch_token):
-    farming_rewards_contract = FarmingRewards.deploy(mooniswap, one_inch_token, rewards_amount, ape, scale, {"from": ape})
-    return farming_rewards_contract
+def farming_rewards(interface):
+    return interface.FarmingRewards(farming_rewards_address)
```

Comment:
Using real contract in tests will broke them after rewards launch

Reply:
Addressed in https://github.com/maddevsio/lido/pull/3/commits/a0f195af475ccf573cde32bedca07f8effc13944